perma_web/perma/models.py

```diff
@@ -215,21 +221,21 @@ def all_folder_trees(self):
         if self.is_registrar_member():
             vesting_orgs = self.registrar.vesting_orgs.all()
         else:
-            vesting_orgs = [self.get_default_vesting_org()]
+            vesting_orgs = list(self.get_default_vesting_org())
+
+
         return [self.root_folder.get_descendants(include_self=True)] + \
             ([vesting_org.shared_folder.get_descendants(include_self=True) for vesting_org in vesting_orgs if vesting_org])
 
     def get_default_vesting_org(self):
```

Comment:
For sure. It's getting awfully crufty in there. I'm out of steam now. Punting this cleanup to the next release with this issue, https://github.com/harvard-lil/perma/issues/876